src/test/java/complete_task_use_case/CompleteTaskDataAccessTest.java

```diff
@@ -0,0 +1,44 @@
+package complete_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessTask;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+/**
+ * Task Completion Data Access Test Cases.
+ * @author Fardin Faruk
+ */
+public class CompleteTaskDataAccessTest {
+    @Test
+    public void testExistsByTitleWithoutDuplicateTask(){
+        CompleteTaskDataAccess DataAccess = new CompleteTaskDataAccess();
```

Comment:
variable should start with a lowercase letter.

Reply:
There are multiple occurences of this in this file.

Reply:
done